packages/flutter_app/ios/Runner/AppDelegate.swift

```diff
@@ -1,7 +1,7 @@
 import UIKit
 import Flutter
 
-@UIApplicationMain
+@main
```

Comment:
![badge](https://img.shields.io/badge/review-ask-informational) 
Is it also necessary to modify `Runner/AppDelegate.swift` in `macos`?

Reply:
Indeed! I committed the macOS to build diff!
https://github.com/altive/flutter_app_template/pull/476/commits/233f6dc5fb28a2113eb72f95621d7025ba79db98